app/src/main/java/org/dicekeys/app/encryption/EncryptedData.kt

```diff
@@ -0,0 +1,32 @@
+package org.dicekeys.app.encryption
+
+import android.util.Base64
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+@Serializable
+data class EncryptedData(
```

Comment:
How is this different from EncryptedDiceKey?  What uses it?

Reply:
EncryptedData is  a generic to store a ciphertext and a IV. That way you can store whatever information you want.

EncryptedDiceKey has the EncryptedData payload as well as more info like keyId and centerFace, that describes the DiceKey.